.vscode/settings.json

```diff
@@ -27,5 +27,6 @@
     "**/bazel-out": true,
     "**/dist": true
   },
-  "peacock.color": "#ff2c70"
+  "peacock.color": "#ff2c70",
+  "terminal.integrated.shell.windows": "C:\\Program Files\\Git\\bin\\bash.exe"
```

Comment:
I think this line should be removed bc it's platform-specific.

Reply:
Oops 😅